### PR TITLE
Python 3.7 builds (fixes cython)

### PR DESCRIPTION
**Pull request by thermokarst**
Checklist
* [x] Used a fork of the feedstock to propose changes
* [x] Bumped the build number (if the version is unchanged)
* [x] [Re-rendered]( https://conda-forge.org/docs/conda_smithy.html#how-to-re-render ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

---

This changeset fixes the linking issue with Python 3.7 and Cython.

cc @wasade @ebolyen

### CODE REVIEW

**Comment by thermokarst** [comment]
@conda-forge-admin, please rerender

File: recipe/meta.yaml [hunk 1]
```diff
@@ -10,10 +10,8 @@ source:
   sha256: b47e54282ef13cddffdb00aea9183a87175a2372c91a915259086a3f444c42f4
 
 build:
-  number: 1001
-  script: python -m pip install --no-deps --ignore-installed .
-  # Cython issues
-  skip: True  # [py>=37]
+  number: 1002
+  script: USE_CYTHON=1 {{ PYTHON }} -m pip install --no-deps --ignore-installed .
```
Comment (nsoranzo) [inline recipe/meta.yaml]: Appveyor builds fail with `'USE_CYTHON' is not recognized as an internal or external command,
operable program or batch file.`, see https://ci.appveyor.com/project/conda-forge/biom-format-feedstock/builds/21535902/job/vauvfd0x6u3xjhnq#L539

**Comment by nsoranzo** [comment]
@thermokarst You also may want to add yourself to https://github.com/conda-forge/biom-format-feedstock/pull/6/files#diff-e178b687b10a71a3348107ae3154e44cR53

**Comment by thermokarst** [comment]
Thanks @nsoranzo - I didn't mean to push [that commit](https://github.com/conda-forge/biom-format-feedstock/pull/6/commits/7303bf23b3482cb85c60cd712777656989a9ac3b) up - still fiddling around here on this end --- will add myself as a ~contributor~ maintainer while I am at it. Thanks!

**Comment by ebolyen** [comment]
Looks like it's working with the exception of a network failure:
https://circleci.com/gh/conda-forge/biom-format-feedstock/88
@nsoranzo, would you be able to restart that one?

**Comment by nsoranzo** [comment]
Green!

**Comment by thermokarst** [comment]
Ready for review/merge - thanks!

File: recipe/meta.yaml [hunk 2]
```diff
@@ -23,7 +20,8 @@ requirements:
   host:
     - python
     - pip
-    - numpy
+    - cython
+    - {{ pin_compatible('numpy') }}
```
Comment (bgruening) [inline recipe/meta.yaml]: this should be `numpy` imho

Comment (thermokarst) [reply recipe/meta.yaml]: Thanks @bgruening! Addressed in https://github.com/conda-forge/biom-format-feedstock/pull/6/commits/a5fa5a662e11998421a095ff774eb02a00cef265.

**Comment by bgruening** [comment]
@conda-forge-admin, please rerender

**Comment by bgruening** [comment]
Thanks all!

**Pull request merged**